flycheck.el

```diff
@@ -2745,14 +2745,21 @@ buffer."
           (flycheck-buffer-automatically 'new-line 'force-deferred)
         (setq flycheck-idle-change-timer
               (run-at-time flycheck-idle-change-delay nil
-                           #'flycheck-handle-idle-change (current-buffer)))))))
+                           #'flycheck--handle-idle-change (current-buffer)))))))
 
-(defun flycheck-handle-idle-change (buffer)
-  "Handle an expired idle timer in BUFFER since the last change."
+(defun flycheck--handle-idle-change (buffer)
```

Comment:
LGTM, except that I would like the wrapper to have a more discerning name than just an added dash.  Maybe add `-internal` or `-in-buffer` to the end?

Reply:
Good point :) Updated.